orc8r/cloud/go/serde/registry.go

```diff
@@ -18,8 +18,29 @@ import (
 	"sort"
 	"sync"
 	"testing"
+
+	"github.com/pkg/errors"
 )
 
+// Registry of serdes, keyed by type.
+type Registry map[string]Serde
```

Comment:
You should create a factory function for Registry that constructs one out of a vararg of Serde inputs. I don't like instantiating these directly, we shouldn't even introduce the opportunity to fat-finger a map key into the API.

A little unfortunate that we have to expose the type and create that footgun anyways. Consider making this an interface that exposes `MustMerge` and `GetMap`.